src/WarcraftLegacies.Source/Quests/Gilneas/QuestCrowley.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MacroTools.ObjectiveSystem.Objectives.TimeBased;
+using MacroTools.QuestSystem;
+using static War3Api.Common;
+
+namespace WarcraftLegacies.Source.Quests.Gilneas
+{
+  /// <summary>
+  /// Darius gets released from Prison " 
+  /// </summary>
+  public sealed class QuestCrowley : QuestData
+  {
+    private const int AltarId = Constants.UNIT_H02X_ALTAR_OF_KINGS_GILNEAS_ALTAR;
+
+    /// <summary>
+    /// IniInitializes a new instance of the <see cref="QuestCrowley"/> class.
+    /// </summary>
+    public QuestCrowley() : base("The Rebel",
+      "Darius Crowley has been imprisoned since the Northgate rebellion, if Gilneas were to be in grave peril an early release might be the solution.",
+      @"ReplaceableTextures/CommandButtons/BTNWorgenHunt.blp")
+    {
+      AddObjective(new ObjectiveTime(900));
+      ResearchId = Constants.UPGRADE_MD05_QUEST_COMPLETED_THE_REBEL;
+    }
```

Comment:
I think you probably want `Required = true` because this is one of Gilneas' basic heroes.